### ISSUE TITLE
The FEN validator doesn't work with the initial position

### ISSUE THREAD

**Issue by FilipRachunek** [yo35/rpb-chessboard#256]
Hello,

First of all, your plugin is one of the most useful tools for WordPress I ever tried. Great work! I use it frequently in my chess blog.

There is just a small issue with the Chess Diagram widget. Sometimes I need to display the starting position with some arrows, etc. However, when I add the diagram to the post draft and click Preview, the preview page shows this error:

```
Error in the FEN string describing the chess diagram.
A FEN string must contain exactly 6 space-separated fields.
```

Would it be possible to fix it, please? The FEN code `rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1` looks pretty valid to me.

Filip

**Comment by yo35** [comment]
This is fixed in version 7.3.10 (just released). Thanks for reporting.

**Comment by FilipRachunek** [comment]
That was quick. Thank you! Works perfectly.
